### ISSUE TITLE
AAAA + IPv6 for https://black.now.sh/

### ISSUE THREAD

**Issue by cooperlees** [jpadilla/black-online#3]
First of all, thanks for the awesome tool for the black demo.

How hard would it be to add IPv6 to the Hostname for us people who use IPv6 only? If it costs extra and you're paying for this yourself I understand.
- Maybe add a donation page? (if it does - I have no idea of aws costs)

Thanks - happy to help with any IPv6 questions,
Cooper

**Comment by jpadilla** [comment]
Ah! Didn't think about this at all...

Custom domains on now require a [premium plan](https://zeit.co/account/plan), not worth for this imho. Might be able to run this on Heroku for free in my account, but will run out of free dyno hours(same thing eventually happened with https://yapf.now.sh).

I could throw it up on AWS with [serverless](https://serverless.com/framework/), that'll probably be next to nothing.

**Comment by cooperlees** [comment]
Yeah, I might just throw your code up on one of my VPS devices too as they are dual stacked :D Many thanks for this and treat it very very very low priority :)

**Comment by jpadilla** [comment]
@cooperlees playing around with the idea of enabling this by deploying to AWS(lambda + api gateway) with [serverless](https://github.com/jpadilla/black-online/tree/serverless). No idea how much that might eventually cost, but worth the experiment for this and other things.

https://black.jpadilla.com
